.github/workflows/tests.yml

```diff
@@ -12,23 +12,21 @@ jobs:
   install:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Use Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: '18'
```

Comment:
According to the [Github documentation](https://github.com/actions/setup-node#:~:text=The%20node%2Dversion%20input%20is%20optional.%20If%20not%20supplied%2C%20the%20node%20version%20from%20PATH%20will%20be%20used.%20However%2C%20it%20is%20recommended%20to%20always%20specify%20Node.js%20version%20and%20don%27t%20rely%20on%20the%20system%20one.), is _recommended to always specify a Node.js version and don't rely on the system one_.